### PR TITLE
changing the run script from nodemon to node

### PR DESCRIPTION
**Pull request by Baalmart**
**_WHAT DOES THIS PR DO?_**

- [x] Updating the run command from nodemon to node for the data-mgt micro (node)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [PLAT-1211]

**_HOW DO I TEST OUT THIS PR?_**
```
cd data-mgt/node
npm install
```
`npm run prod-mac` OR `npm run prod-pc`

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
Any

- [ ] [Get most recent descriptive measurements](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/data-management#get-last-entry-descriptive)

[PLAT-1211]: https://airqoteam.atlassian.net/browse/PLAT-1211?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/data-mgt/node/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "private": true,
     "main": "./bin/www",
     "scripts": {
-        "start": "nodemon ./bin/www",
+        "start": "node ./bin/www",
```
Comment (Baalmart) [inline src/data-mgt/node/package.json:7]: just changed from `nodemon` to `node` in the start scripts

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @Baalmart 
LGTM

File: src/data-mgt/node/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
         "dev-mac": "NODE_ENV=development nodemon ./bin/www",
         "prod-mac": "NODE_ENV=production nodemon ./bin/www",
         "stage-mac": "NODE_ENV=staging nodemon ./bin/www",
```
[the remark this answers is not in the document]
Comment (Baalmart) [reply src/data-mgt/node/package.json:10]: Thanks @NoahNsimbe for this observation.

I guess it is just for development purposes when testing the application on local machine. `nodemon` just enables automatic restarts (hot reloads) during development/testing process.

As for the default run command (production), it is recommended to just use `node` in production mode.

[the remark this answers is not in the document]
Comment (Baalmart) [reply src/data-mgt/node/package.json:10]: @NoahNsimbe , awaiting your additional thoughts on this. Thanks.

[the remark this answers is not in the document]
Comment (Baalmart) [reply src/data-mgt/node/package.json:10]: Hi @NoahNsimbe, I am just merging this since there is no comment from you.

**Pull request merged**